src/components/tipRecords/AuthorAndDate.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <div
+    class="author-and-date"
+    :title="address"
+    @click.stop
+  >
+    <router-link
+      :to="{
+        name: 'user-profile',
+        params: {
+          address,
+        },
+      }"
+    >
+      <Avatar :address="address" />
+      <div class="author-name">
+        <span
+          v-if="chainName"
+          class="chain-name"
+        >
+          {{ chainName }}
+        </span>
+        <span
+          v-if="!chainName || (tipRecord && chainName)"
+          class="address"
+        >
+          {{ address }}
+        </span>
+      </div>
+    </router-link>
+    <span class="date">
```

Comment:
```suggestion
    <span class="right">
```
🤷‍♀️because it contains not only the date